### PR TITLE
fix stopCheckReady receive signal hang for status update.

### PR DESCRIPTION
**Pull request by morvencao**
fix: https://github.com/open-cluster-management/backlog/issues/15667

Signed-off-by: morvencao <lcao@redhat.com>

### CODE REVIEW

File: operators/multiclusterobservability/controllers/multiclusterobservability/multiclusterobservability_status.go [hunk 1]
```diff
@@ -39,19 +39,22 @@ func StartStatusUpdate(c client.Client, instance *mcov1beta2.MultiClusterObserva
...
 					updateStatus(c)
 					if updateReadyStatusIsRunnning && checkReadyStatus(c, instance) {
+						log.V(1).Info("send singal to stop status check ready goroutine because MCO status is ready")
 						stopCheckReady <- struct{}{}
 					}
 				}
 			}
 		}()
 		if !updateReadyStatusIsRunnning {
+			// init the stop ready chaek channel
```
Comment (songleo) [inline operators/multiclusterobservability/controllers/multiclusterobservability/multiclusterobservability_status.go:56]: have a typo error

Comment (morvencao) [reply operators/multiclusterobservability/controllers/multiclusterobservability/multiclusterobservability_status.go]: nice catch! fixed.

**Pull request merged**